hope.yaml

```diff
@@ -393,6 +397,42 @@ resources:
         name: mysql-backup
   - name: mysql
     file: mysql/mysql.yaml
+  - name: mongodb-block-storage
+    file: mongodb/mongodb-block-storage.yaml
+  - name: mongodb-backup
+    inline: |
+      apiVersion: v1
```

Comment:
Similar to the mysql one, probably worth not embedding this in hope.yaml, and just leaving it in its own file. 

Possibly worth creating a mechanism in hope to create configmaps/secrets from files on disk, similar to the kubectl commands. That way the file still stands on its own as a script, and can take advantage of syntax highlighting and the like.